import_musicbrainz_row_ids.py

```diff
@@ -0,0 +1,115 @@
+from listenbrainz import db
+from brainzutils import musicbrainz_db
+from listenbrainz.webserver import create_app
+from listenbrainz.webserver.views.api_tools import publish_data_to_queue
+from listenbrainz.listenstore import InfluxListenStore
+from listenbrainz.webserver.influx_connection import init_influx_connection
+from werkzeug.exceptions import NotFound
+
+import listenbrainz.db.user as db_user
+import logging
+import sqlalchemy
+
+app = create_app()
+influx = init_influx_connection(logging, {
+            'REDIS_HOST': app.config['REDIS_HOST'],
+            'REDIS_PORT': app.config['REDIS_PORT'],
+            'REDIS_NAMESPACE': app.config['REDIS_NAMESPACE'],
+            'INFLUX_HOST': app.config['INFLUX_HOST'],
+            'INFLUX_PORT': app.config['INFLUX_PORT'],
+            'INFLUX_DB_NAME': app.config['INFLUX_DB_NAME'],
+        })
+
+
+def fix_username_for_exceptions():
+    with db.engine.connect() as connection:
+        # 2106 - Fée Deuspi
+        connection.execute(sqlalchemy.text("""
+            UPDATE "user"
+               SET musicbrainz_id = 'Fée Deuspi'
+             WHERE id = 2106
+            """))
+
+        # 243 - ClæpsHydra
+        connection.execute(sqlalchemy.text("""
+            UPDATE "user"
+               SET musicbrainz_id = 'ClæpsHydra'
+             WHERE id = 243
+            """))
+
+
+def delete_user(user):
+    influx.delete(user['musicbrainz_id'])
+    publish_data_to_queue(
+        data={
+            'type': 'delete.user',
+            'musicbrainz_id': user['musicbrainz_id'],
+        },
+        exchange=app.config['BIGQUERY_EXCHANGE'],
+        queue=app.config['BIGQUERY_QUEUE'],
+        error_msg='Could not upt user %s into queue for bq deletion.' % user['musicbrainz_id'],
+    )
+    db_user.delete(user['id'])
+
+
+def import_musicbrainz_rows(musicbrainz_db_uri, dry_run=True):
```

Comment:
Normally I would say that we should be catching exceptions here, but given that this script will only ever be run by you and hopefully only once, I suppose we can skip it.